Cargo.toml

```diff
@@ -44,8 +44,9 @@ dirs = "4.0"
 bs58 = "0.4"
 rand = "0.8"
 aes = "0.7"
+rocksdb = "0.20.1"
 
-dusk-wallet-core = "0.15.0-rc"
+dusk-wallet-core = { version = "0.15.0-rc.1", git = "https://github.com/dusk-network/wallet-core", branch = "fetch_notes_cache_fix" }
```

Comment:
Git dependencies should be removed when merging in main

Reply:
After discuss with @ZER0 we believe it's not worth to have a bump in wallet_core right now.
So we should go back to the initial implementation where the key of notes is the hash and not the nullifier.

Going to open an issue to explain why the nullifier is still required (future features) and why we choose to not implement right now (wallet_core will be removed)